torchbenchmark/models/Super_SloMo/slomo_model.py

```diff
@@ -244,11 +244,10 @@ def __init__(self, W, H, device):
 
         super(backWarp, self).__init__()
         # create a grid
-        gridX, gridY = np.meshgrid(np.arange(W), np.arange(H))
         self.W = W
         self.H = H
-        self.gridX = torch.tensor(gridX, requires_grad=False, device=device)
-        self.gridY = torch.tensor(gridY, requires_grad=False, device=device)
+        self.gridX, self.gridY = torch.meshgrid(torch.arange(W, requires_grad=False, device=device), 
```

Comment:
Can we add a comment here referring to the PR to the upstream repo?

Reply:
Comment added.